python/paddle/fluid/tests/unittests/mkldnn/test_scale_mkldnn_op.py

```diff
@@ -77,6 +87,7 @@ def test_check_grad(self):
         self.check_grad(['X'], 'Out')
 
 
+# FIXME(xx) no use_mkldnn attr, does this case run into oneDNN?
```

Comment:
Please remove this comment. It looks like: "I will never fix it".

Reply:
fixed